devices/Mpu6886/Mpu6886.cs

```diff
@@ -0,0 +1,371 @@
+        public Vector3 GetGyroscopeOffset()
+        {
+            SpanByte vec = new byte[6];
+            Read(Mpu6886.Register.GyroscopeOffsetAdjustmentXHighByte, vec);
+
+            Vector3 v = new Vector3();
+            v.X = (short)(vec[0] << 8 | vec[1]);
+            v.Y = (short)(vec[2] << 8 | vec[3]);
+            v.Z = (short)(vec[4] << 8 | vec[5]);
+
+            return v;
+        }
+
+        /// <summary>
+        /// Reset the internal registers and restores the default settings. (Datasheet, page 47)
+        /// </summary>
+        public void Reset()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b1000_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Set the chip to sleep mode. (Datasheet, page 47)
+        /// </summary>
+        public void Sleep()
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement1, 0b0100_0000 }));
+            Thread.Sleep(10);
+        }
+
+        /// <summary>
+        /// Sets the accelerometer full scale. (Datasheet page 37)
+        /// </summary>
+        /// <param name="scale">The scale to set.</param>
+        public void SetAccelerometerScale(AccelerometerScale scale)
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.AccelerometerConfiguration1, (byte)scale }));
+            Thread.Sleep(1);
+        }
+
+        /// <summary>
+        /// Sets the gyroscope full scale. (Datasheet page 37)
+        /// </summary>
+        /// <param name="scale">The scale to set.</param>
+        public void SetGyroscopeScale(GyroscopeScale scale)
+        {
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.GyroscopeConfiguration, (byte)scale }));
+            Thread.Sleep(1);
+        }
+
+        /// <summary>
+        /// Sets the enabled axes of the gyroscope and accelerometer. (Datasheet page 47)
+        /// </summary>
+        public EnabledAxis EnabledAxes
+        {
+            get
+            {
+                SpanByte readBuffer = new byte[1];
+                _i2c.WriteByte((byte)Mpu6886.Register.PowerManagement2);
+                _i2c.Read(readBuffer);
+
+                // bit 1 in the register means disabled, so using bitwise not to flip bits.
+                return (EnabledAxis)~readBuffer[0];
+            }
+            set
+            {
+                // bit 1 in the register means disabled, so using bitwise not to flip bits.
+                _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.PowerManagement2, (byte)~value }));
+                Thread.Sleep(1);
+            }
+        }
+
+        /// <summary>
+        /// Sets the averaging filter settings for low power accelerometer mode. (Datasheet page 37)
+        /// </summary>
+        /// <param name="mode">The mode to set.</param>
+        public void SetAcceleratorLowPowerMode(AccelerometerLowPowerMode mode)
```

Comment:
As you can as well read the AccelerometerLowPowerMode, I would transform this one as well as a property